lib/rucio/core/replica.py

```diff
@@ -1486,9 +1537,15 @@ def __bulk_add_replicas(rse_id, files, account, *, session: "Session"):
     for f in files:
         condition.append(and_(models.RSEFileAssociation.scope == f['scope'], models.RSEFileAssociation.name == f['name'], models.RSEFileAssociation.rse_id == rse_id))
 
-    query = session.query(models.RSEFileAssociation.scope, models.RSEFileAssociation.name, models.RSEFileAssociation.rse_id).\
-        with_hint(models.RSEFileAssociation, text="INDEX(REPLICAS REPLICAS_PK)", dialect_name='oracle').\
-        filter(or_(*condition))
+    stmt = select(
+        models.RSEFileAssociation.scope,
+        models.RSEFileAssociation.name,
+        models.RSEFileAssociation.rse_id,
+    ).where(
+        or_(*condition)
+    ).with_hint(models.RSEFileAssociation, "INDEX(REPLICAS REPLICAS_PK)", 'oracle')
+
+    query = session.execute(stmt)
```

Comment:
it's not a `query` anymore, it's a `result`.

Reply:
Renamed this and one other occurrence of this error in new commit